src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/ReissuanceConsent/HealthCertifiedPersonReissuanceConsentViewController.swift

```diff
@@ -31,10 +41,12 @@ class HealthCertifiedPersonReissuanceConsentViewController: UIViewController, Di
 
 	override func viewDidLoad() {
```

Comment:
We need to mark the `certificateReissuance` as read on viewDidAppear (like we do for the booster notification in `BoosterDetailsViewController`). In this case we need to set `isNewCertificateReissuance` on the person to `false`